### PR TITLE
gwt: remove bottle :unneeded.

### PR DESCRIPTION
**Pull request by cho-m**
https://github.com/Homebrew/homebrew-core/issues/75943

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Extracted from #79997 to work on separately

### CODE REVIEW

**Comment by yurikoles** [comment]
It obviously misses `openjdk` dependency.

**Comment by cho-m** [comment]
@yurikoles Yes.

It also requires updating all the scripts so they can find JAVA_HOME.
I just haven't had the chance to go through it yet.
Some of them may be better changed to `write_env_script`.

File: Formula/gwt.rb [hunk 1]
```diff
@@ -10,31 +10,14 @@ class Gwt < Formula
...
-      #!/bin/sh
-      HOMEDIR=#{libexec}
-      java -cp "$HOMEDIR/gwt-user.jar:$HOMEDIR/gwt-dev.jar" com.google.gwt.user.tools.WebAppCreator "$@";
-    EOS
-
-    (bin/"benchmarkViewer").write <<~EOS
-      #!/bin/sh
-      APPDIR=#{libexec}
-      java -Dcom.google.gwt.junit.reportPath="$1" -cp "$APPDIR/gwt-dev.jar" com.google.gwt.dev.RunWebApp -port auto $APPDIR/gwt-benchmark-viewer.war;
-    EOS
```
Comment (cho-m) [inline Formula/gwt.rb:31]: Not sure if this still exists: `#{libexec}/gwt-benchmark-viewer.war`

```console
❯ benchmarkViewer
Could not open war file '/usr/local/Cellar/gwt/2.9.0/libexec/gwt-benchmark-viewer.war'
Google Web Toolkit 2.9.0

❯ fd -IHe war

❯ fd -IHe jar
gwt-api-checker.jar
gwt-codeserver.jar
...

❯ fd benchmark

❯ fd i18nCreator
i18nCreator
i18nCreator.cmd

❯ fd webAppCreator
webAppCreator
webAppCreator.cmd
```

**Comment by cho-m** [comment]
Checksums all match:
```
aa7923025737fa93944f4373b92544bacdc5e191b679f1b615b65b0455c2e7a8  gwt--2.9.0.arm64_big_sur.bottle.tar.gz
aa7923025737fa93944f4373b92544bacdc5e191b679f1b615b65b0455c2e7a8  gwt--2.9.0.big_sur.bottle.tar.gz
aa7923025737fa93944f4373b92544bacdc5e191b679f1b615b65b0455c2e7a8  gwt--2.9.0.catalina.bottle.tar.gz
aa7923025737fa93944f4373b92544bacdc5e191b679f1b615b65b0455c2e7a8  gwt--2.9.0.mojave.bottle.tar.gz
```

Just need review on script changes and should be good to merge.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**